pkg/monitor/payload/monitor_payload.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2017 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package payload
+
+import (
+	"bytes"
+	"encoding/gob"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+const (
+	// EventSample is equivalent to PERF_RECORD_SAMPLE
```

Comment:
`is equivalent to PERF_RECORD_SAMPLE` it would help to know from which file is this on